Updated JiraV2 integration to combine issueJson with the transition a… (#19734)
demisto/content: Packs/Jira/ReleaseNotes/2_0_5.md

#### Integrations
##### Atlassian Jira v2

> - Updated integration to combine issueJson with the transition and status, instead of doing it before. This makes it possible to fill out the resolution field while doing a transition to closed status

Reviewer: ```suggestion
- You can now fill modify the resolution field while transitioning a status to closed.
```
